frontend/components/Storyline/StorylineOverview/StorylineOverviewCard.tsx

```diff
@@ -1,57 +1,82 @@
 /* eslint-disable @next/next/no-img-element */
 import { BoltIcon, InformationCircleIcon, DocumentTextIcon } from "@heroicons/react/24/outline";
-import RawHtml from "../RawHtml";
+import RawHtml from "../../RawHtml";
+import Image from "next/image";
+import Link from "next/link";
+import { useRouter } from "next/router";
 
 interface Props {
-  key: number;
+  index: number;
   project: {
```

Comment:
Ideally, we'd have a standard `Project` type somewhere that we can reuse.

`StorylineProps` and this `project` type have a lot in common, and in `StorylineOverview` we're treating them as the same (by passing a `StorylineProps` into `StorylineOverviewCard` as the `project` parameter.

Reply:
I agree and have improved it.